fhir-datastore-hapi-fhir/docker-compose-postgres.yml

```diff
@@ -13,6 +13,7 @@ services:
       REPMGR_NODE_NAME: postgres-1
```

Comment:
Good time to name this like the convention we've decided... `postgres-01`

Reply:
Cool done :)

Reply:
I just checked now, found out that most of the services are -1 and not -01, what about updating all to be -1 and -2 instead ?

Reply:
Hmmm, neither here nor there for me, as long as it is consistent. All I know is that, when we introduced JeMPI, we decided on -01, -02, etc...

@michaelloosen , please advise.

Reply:
Yeah, also neither here nor there, but I think it was @tumbledwyer who decided on the 01, 02 convention, so let's get his comment in here...

Reply:
cool I will apply this fix on another PR as it is going to affect a lot of packages

Reply:
Let's make it with the 0 in front of the number so that we can expand to 10 without it being weird. 👍